File: myutils/src/main/java/com/jty/myutils/constant/MemoryUtils.java
package com.jty.myutils.constant;

/**
 * @author TaoYuan
 * @time 2017/3/22 0022
 * @desc
 */

public class MemoryUtils {
    /******************** 存储相关常量 ********************/
    /**
     * Byte与Byte的倍数
     */
    public static final int BYTE = 1;
    /**
     * KB与Byte的倍数
     */
    public static final int KB   = 1024;
    /**
     * MB与Byte的倍数
     */
    public static final int MB   = 1048576;
    /**
     * GB与Byte的倍数
     */
    public static final int GB   = 1073741824;

    public enum MemoryUnit {
        BYTE,
        KB,
        MB,
        GB
    }
}

File: mykotlinutils/src/main/java/com/jty/mykotlinutils/constant/MemoryUtils.kt
package com.jty.kotlin.constant

/**
 * Created by TaoYuan on 2017/9/20 0020.
 *
 * Stored Constant
 */

object MemoryUtils {

    /** BYTE */
    const val BYTE = 1

    /** BYTE * 1024 */
    const val KB = 1024

    /** KB * 1024 */
    const val MB = 1048576

    /** MB * 1024 */
    const val GB = 1073741824

    /** Stored Unit */
    enum class MemoryUnit {
        BYTE,
        KB,
        MB,
        GB
    }
}
